docs/guides/tutorials/logging_tutorial.md

```diff
@@ -395,7 +382,7 @@ for exp in experiments:
     careamist.train(train_source=train_data)
 
 # View all experiments together
-# tensorboard --logdir /scratch/username/tensorboard_runs/n2v_comparison
+tensorboard --logdir /path_to_folders/tensorboard_runs/n2v_comparison
```

Comment:
Are you sure about the `tensorboard logdir` rather than `tensorboard --logdir="..."?